core/src/main/java/org/apache/accumulo/core/logging/CorrelationIdLogger.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.accumulo.core.logging;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.slf4j.event.Level;
```

Comment:
I'm still a bit uncomfortable more tightly coupling ourselves to more SLF4J APIs, and have reservations about using Level here. Log4j's Level caused us a lot of problems removing it from our code, and although SLF4J's Level is maybe situated differently in its API, I can't help but anticipate the potential future problems we could have by using more of the logging framework's APIs in our code. I'd really prefer to avoid it. I also think that what I suggested before about having a logging lifecycle class, like what Keith did elsewhere would be better than adding this Logger class. I know Keith said that was out of scope for this PR... but I'm not so sure. If having that avoids this new class I think that's better than adding this class. If it needs to be done first, so it's a discrete task, that's fine... but I do think it's better than adding this class.

Reply:
I do agree with removing the extra sl4j new dependencies - I think the proposed solution of using Keith's method is what we should be targeting.  

As Christopher noted, the change in logging because of dependency changes took quite a while to unwind because we had basically painted ourselves into a corner, avoiding that happening again would be better.

I think those changes could happen first, or this can be committed with the understanding that it would be reworked before any release.  Separate PRs would make the review easier and either way would set-up merge conflicts with this PR.